src/ja/server/database/types/work_machine.py

```diff
@@ -0,0 +1,115 @@
+from ja.common.work_machine import ResourceAllocation
+
+
+class RunningWorkMachineResources:
```

Comment:
Why did you call this class RunningWorkMachineResources and not just something like WorkerResources?

Reply:
Renaming it to WorkMachineResources (I don't think we use Worker much)